app/code/core/Mage/Catalog/Model/Category.php

```diff
@@ -314,7 +314,7 @@ public function move($parentId, $afterCategoryId)
     /**
      * Retrieve default attribute set id
      *
-     * @return int
+     * @return string
```

Comment:
Attribute set ID is stored as int in the DB. Is it more correct to return as int? I'm aware that reading int from DB converts to string. It's problematic. But in the spirit of strict typing, it should be int. Your thought?

Reply:
Yes. Same as for "store_id". Cast to int?